Source/WebInspectorUI/UserInterface/Views/RecordingActionTreeElement.js

```diff
@@ -101,6 +105,7 @@ WI.RecordingActionTreeElement = class RecordingActionTreeElement extends WI.Gene
             case WI.Recording.Swizzle.WebGLVertexArrayObject:
                 parameterElement.classList.add("swizzled");
                 parameterElement.textContent = WI.Recording.displayNameForSwizzleType(swizzleType);
+                parameterElement.textContent += ": " + parameter;
```

Comment:
We should check to make sure that there is a `parameter` value before attempting to use it, as using a newer Web Inspector with an older target (e.g. an iPhone running an older iOS) or a WebGL recording created with an older Web Inspector will have problems with this.
```
parameterElement.textContent = WI.Recording.displayNameForSwizzleType(swizzleType) + (parameter ? (": " + parameter) : "");
```

Reply:
based on my above comment maybe we should have it be
```
parameterElement.textContent = WI.Recording.displayNameForSwizzleType(swizzleType) + (parameter ? (`: (@${parameter})`) : "");
```

Reply:
Currently, all WebGL objects are recorded with a parameter value of `0`.

Reply:
> based on my above comment maybe we should have it be
> 
> ```
> parameterElement.textContent = WI.Recording.displayNameForSwizzleType(swizzleType) + (parameter ? (`: (@${parameter})`) : "");
> ```

I've implemented this and pushed.

Reply:
something I forgot to mention, please make sure that this extra text is not included if the user copies the selected action (see the `copyText` variable inside the `WI.RecordingActionTreeElement.prototype._generateDOM`)